catala/cantar/tastar/tastar.md

```diff
@@ -0,0 +1 @@
+Tastes una llaminadura en forma de cor, i a fora sembla que para de ploure. Obres la tenda i... tot és de sucre!
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.